Slack-Apps/week3/homework/responses/3.3-calling-translator-api.md

```diff
@@ -0,0 +1,90 @@
+# Google Translate API: Calling the Translator
+
+You should now return to your ```app.js``` to implement adding Cloud Translation. In the livestream, you should have noticed the Slack App response was hardcoded to a phrase like "This is Spanish" or "This is Japanese". Now, we are going to be adding the response as the actual ```translated text``` in a thread.
+
+## Integrating the API
+
+In the first issue (3.1), you should have already used ```.includes``` and ```.split(-)``` to find the country code of the flag and if not, created a langMap. It should have looked something like this: ```const [x, country] = langMap[reaction] ? [, reaction] : reaction.split('-');```
+
+We also added the feature of responding to a thread but which thread you may ask? Well, we need to find that. To do so, we are going to set ```const result``` equal to ```await client.conversations.history({ Edit Here })```.
+
+## Country Code
+
+Now that we have the ```const country``` set to our country code, we need to find the corresponding language to our country code. To do this, we need to create a new variable that is equal to the corresponding language. To do this, we can iterate through or simply search through our langMap for the language (*remember, it has two arguments: country and language which we need to set equal to language*).
+
```

Comment:
maybe just a personal preference but I think some would benefit from seeing the `langMap` beside the line of code below just to visualize what values are being referenced.